pkg/cmd/cmd_add.go

```diff
@@ -91,5 +90,10 @@ func onlyOnceOption(c *cli.Context, name string) (*string, error) {
 }
 
 func addGitDep(opt *opt.AddOptions, kclPkg *pkg.KclPkg) error {
-	return ops.KpmAdd(opt, kclPkg)
+	if opt.RegistryOpts.Git == nil {
```

Comment:
There is usually a unified method for checking these parameters. For example:
https://github.com/KusionStack/kusion/blob/main/pkg/cmd/init/options.go#L53

Reply:
fixed.